src/mpid/ch4/src/ch4r_proc.h

```diff
@@ -204,7 +204,7 @@ MPL_STATIC_INLINE_PREFIX int MPIDIU_comm_rank_to_pid_local(MPIR_Comm * comm, int
 
 MPL_STATIC_INLINE_PREFIX int MPIDI_CH4U_rank_is_local(int rank, MPIR_Comm * comm)
 {
-    int ret;
+    int ret = 0;
     MPIR_FUNC_VERBOSE_STATE_DECL(MPID_STATE_MPIDI_CH4U_RANK_IS_LOCAL);
     MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_MPIDI_CH4U_RANK_IS_LOCAL);
 
```

Comment:
You may take `ret = 0` out later in `if (comm->comm_kind == MPIR_COMM_KIND__INTERCOMM)` condition. (commenting here because I couldn't write there)